# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/memtest86/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./test.h.orig	2002-08-31 18:47:01.000000000 +0200
+++ ./test.h	2002-08-31 18:47:13.000000000 +0200
@@ -277,7 +277,6 @@
 #define FIRMWARE_PCBIOS    1
 #define FIRMWARE_LINUXBIOS 2
 
-extern struct vars * const v;
 extern unsigned char _start[], _end[], startup_32[];
 extern unsigned char _size, _pages;
 
--- ./memsize.c.orig	2002-08-31 18:47:20.000000000 +0200
+++ ./memsize.c	2002-08-31 18:47:53.000000000 +0200
@@ -18,6 +18,7 @@
 
 extern ulong p1, p2;
 extern volatile ulong *p;
+extern struct vars *v;
 
 static void sort_pmap(void);
 static int check_ram(void);
